tests/test_views.py

```diff
@@ -116,5 +122,49 @@ def test_response_404_when_none_specified(self):
             content = json.loads(response.content.decode('utf-8'))
             self.assertCountEqual({'message': 'No checks found', 'error': 404}, content)
 
+    @override_settings(WATCHMAN_LOGIN=False)
+    def test_login_not_required(self):
+        # Reload settings - and all dependent modules - from scratch
+        reload(sys.modules['watchman.settings'])
+        reload(sys.modules['watchman.decorators'])
+        reload(sys.modules['watchman.views'])
```

Comment:
Does it make sense to move these statements into a small `reload_watchman_modules` utility function since we're doing the same thing over and over in these tests?

Secondly, I'm curious as to why this is necessary - is it to pick up the `@override_settings` decorator you're using? Or is it just to guarantee that settings aren't being mixed and matched between tests?